frontend/src/pages/org/index.ts

```diff
@@ -197,7 +193,22 @@ export class Org extends LiteElement {
         void this.updateOrg();
       } else {
         // Couldn't find org with slug, redirect to first org
-        this.navTo(`/orgs/${this.userInfo.orgs[0].slug}`);
+        const org = this.userInfo.orgs[0] as UserOrg | undefined;
+        if (org) {
+          this.navTo(`/orgs/${org.slug}`);
+        } else {
+          // Handle edge case where user does not belong
+          // to any orgs but is attempting to log in
+          // TODO check if hosted instance and show support email if so
+          this.notify({
+            message: msg(
+              "You must belong to at least one org in order to log in. Please contact your Browsertrix admin to resolve the issue.",
```

Comment:
How do I contact them???  Could / should we link the support email here?  Perhaps a toast is not the best place for this :\

Reply:
This should be an edge case, and per the TODO comment if this becomes a more common issue, we can work on this once the frontend is able to differentiate the hosted instance.